CUJExecutor/dataAggregation/Utility.java

```diff
@@ -0,0 +1,236 @@
+package dataAggregation;
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.util.Scanner; // Import the Scanner class to read text files
+import java.util.*;
+import java.io.*;
+
+
+class Utility {
+
+	/**
+	 * Provides the info necessary to identify a section of a video (a clip)
+	 */
+   static class clipInfo {
+	String folder; //the folder where the clip is located (should be called "full_video.mp4)
+	String start; //the time at which this clip begins in the video (in the format HH:MM:SS.sss)
+	String end; //the time at which this clip ends in the video (in the format HH:MM:SS.sss)
+
+	public clipInfo (String fldr, String timestamps) {
```

Comment:
fldr is a poor variable name.  Should just use folder, then assign it to a field with:

    this.folder = folder